analysis/qc-checks/plots_for_rst_pilot_project_QC.R

```diff
@@ -0,0 +1,391 @@
+library(tidyverse)
+library(lubridate)
+library(googleCloudStorageR)
+library(wakefield)
```

Comment:
note that when wakefield is loaded it seems to mess up lubridate functions like year(date)